lib/web/ui/server.go

```diff
@@ -78,8 +78,27 @@ func (s sortedLabels) Len() int {
 	return len(s)
 }
 
+// sendToBackOfSortNames are label names that we want to always be at the end of
+// the sorted labels list to reduce visual clutter. This will generally be automatically
+// discovered cloud provider labels such as azure/aks-managed-createOperationID=123123123123
+var sendToBackOfSortNames = append([]string{}, types.CloudLabelPrefixes...)
+
 func (s sortedLabels) Less(i, j int) bool {
-	return s[i].Name < s[j].Name
+	labelA := strings.ToLower(s[i].Name)
+	labelB := strings.ToLower(s[j].Name)
+
+	for _, sortName := range sendToBackOfSortNames {
```

Comment:
```suggestion
	for _, sortName := range types.CloudLabelPrefixes {
```

Why not just do this and remove the separate var and the confusing append call?

Reply:
Yeah, makes sense